health/notifications/discord/metadata.yaml

```diff
@@ -0,0 +1,72 @@
+# yamllint disable rule:line-length
+---
+- id: 'notify-discord'
+  meta:
+    name: 'Discord'
+    link: 'https://discord.com/'
+    categories:
+      - notify.agent
+    icon_filename: 'discord.png'
+  keywords:
+    - Discord
+  overview:
+    notification_description: |
+      Send notifications to Discord using Netdata's Agent alert notification feature, which supports dozens of endpoints, user roles, and more.
+    notification_limitations: ''
+  setup:
+    prerequisites:
+      list:
+        - title: ''
+          description: |
+            - The incoming webhook URL as given by Discord. Create a webhook by following the official [Discord documentation](https://support.discord.com/hc/en-us/articles/228383668-Intro-to-Webhooks). You can use the same on all your Netdata servers (or you can have multiple if you like - your decision).
+            - One or more Discord channels to post the messages to
+            - Access to the terminal where Netdata Agent is running
+    configuration:
+      file:
+        name: 'health_alarm_notify.conf'
+      options:
+        description: 'The following options can be defined for this notification'
+        folding:
+          title: 'Config Options'
+          enabled: true
+        list:
+          - name: 'SEND_DISCORD'
+            default_value: ''
+            description: "Set `SEND_DISCORD` to YES"
+            required: true
+          - name: 'DISCORD_WEBHOOK_URL'
+            default_value: ''
+            description: "set `DISCORD_WEBHOOK_URL` to your webhook URL."
+            required: true
+          - name: 'DEFAULT_RECIPIENT_DISCORD'
```

Comment:
I think the `required` field is missing for this.

Also I think in general we should have the `DEFAULT_RECIPIENT_XXXX` as `required: true` ?